mlbench/worker/main.py

```diff
@@ -1 +1,148 @@
-print("Finished")
+import argparse
+import torch
+import os
+import torch.distributed as dist
+
+
+def get_hostname():
+    with open('/etc/hostname', 'r') as f:
```

Comment:
Using 
`import socket`
`print(socket.gethostname())`

or an environment variable ( https://kubernetes.io/docs/concepts/containers/container-environment-variables/ ) is better than /etc/hostname, since it should work no matter what OS the docker file uses.